src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export class Footer extends Component {
+  render() {
+    return (
+      <div>Footer</div>
+    )
+  }
+}
+
+export default Footer
```

Comment:
You have two exports on each of your pages and components which may throw some errors.  Would recommend removing one on each page/component.